addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -1238,8 +1238,10 @@ export class OdooEditor extends EventTarget {
                     focusNode,
                     computedSelection.focusOffset,
                 );
+                return true;
             }
         }
+        return false;
```

Comment:
Wait so tests like this one actually never ever worked ? https://github.com/odoo/odoo/pull/101768/files#diff-aa4686cd9f6a0d50e1040ba3dda0b29e9898e7cc31a47f1f02b46a2c4c047a72R980

Reply:
I guess so yes...
It shocked me too.